tasks/centos.yml

```diff
@@ -1,13 +1,36 @@
+- name: "Install Storage SIG Yum Repos"
+  yum: >
+   name="centos-release-gluster{{ gluster_version }}"
+  tags:
+    - glusterfs
+
+- name: "Install GlusterFS"
+  yum: >
+   name={{ item }}
+   state=present
+   enablerepo=centos-gluster{{ gluster_version }}
```

Comment:
Can't we have a `set_fact` to define the repo name instead of having two similar blocks?